debian/control

```diff
@@ -2,58 +2,56 @@ Source: swtpm
 Maintainer: Stefan Berger <stefanb@linux.vnet.ibm.com>
 Section: misc
 Priority: optional
-Build-Depends: automake,
-	       autoconf,
```

Comment:
These dependencies are transitively implied by `debhelper` (through `dh-autoreconf`) and `libtool`.